WooCommerce/Classes/POS/ViewModels/PointOfSaleDashboardViewModel.swift

```diff
@@ -36,6 +36,22 @@ final class PointOfSaleDashboardViewModel: ObservableObject {
 
     private var cancellables: Set<AnyCancellable> = []
 
+    @Published var showSimpleProductsModal: Bool = false
+
+    @Published var activeModal: ModalType? = nil
+
+    enum ModalType: Identifiable {
+        case simpleProducts
+        // Add other modals if needed
+
+        var id: String {
+            switch self {
+            case .simpleProducts:
```

Comment:
Good thing there is no default case. When we add new cases the compiler will show us where we need to add some logic :) p2: pfoUAQ-gM-p2